Tests/TuistKitAcceptanceTests/LintAcceptanceTests.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import TuistAcceptanceTesting
+@testable import TuistKit
+
+final class LintAcceptanceTests: TuistAcceptanceTestCase {
+    func test_ios_app_with_headers() async throws {
+        try await setUpFixture(.iosAppWithHeaders)
+        try await run(LintImplicitImportsCommand.self)
+        XCTAssertStandardOutput(pattern: "We did not find any implicit dependencies in your project.")
+    }
+
+    func test_ios_app_with_implicit_dependencies() async throws {
+        try await setUpFixture(.iosAppWithImplicitDependencies)
+        try await run(LintImplicitImportsCommand.self)
```

Comment:
I'd expect this command to error out if it finds implicit imports.

Reply:
I can understand why, but also somewhy i dont want to mark it as error. Because command works good and its not an error xd

Reply:
This should be an error. The same way linters usually return a non-zero exit code if they find critical issues in your code.

Reply:
Yes, you will need to throw an error 🙂

Reply:
I dont understand what you want me to do. In case of error it would look like that.
<img width="968" alt="image" src="https://github.com/user-attachments/assets/9907b4aa-c546-4933-8c5b-0cc31e5af5f4">
We can print Lint errors as error description as well, but all that text is not suit this case.

> If you think it's a legit issue, please file an issue including

Reply:
Now it looks like that
<img width="736" alt="Screenshot 2024-08-20 at 00 16 49" src="https://github.com/user-attachments/assets/459c3de0-0af5-4e50-b9ae-21b3fdc75624">

Reply:
@pepicrft @fortmarek  is that okay?

Reply:
@pepicrft What do you think?

Reply:
Added --strict flag now, it can be presented as an error and as an warning(Helpful when you want to present it in xcode for example)